base/setup/usetup/bootsup.c

```diff
@@ -888,7 +888,6 @@ InstallFat16BootCodeToFile(
         return Status;
     }
 
-    FileOffset.QuadPart = 0ULL;
```

Comment:
This is not redundant. It creates clarity and makes the code snippet immune to changes in previous operations in the same function.

Reply:
Did you mean to remove the unused assignment before NtWriteFile below instead?

Reply:
Reverted.
`NtWriteFile` will be part 4/4.

Reply:
Doesn't look reverted?

Reply:
There are 2 of them: I reverted redundant line 891 as asked to, I left unused line 1004.

Reply:
I assumed the line that I commented one was the one that needed changing. Let me just re-review this function instead.

Reply:
You're right, this change is fine.